src/expression.d

```diff
@@ -15915,6 +15920,16 @@ extern (C++) final class FileInitExp : DefaultInitExp
 
 /***********************************************************
  */
+extern (C++) final class FileFullPathInitExp : FileInitExp
```

Comment:
I'd still like to see this class go away and just use `FileInitExp` with a different `op`. It won't be the first AST node that can have more than one value for `op`.